docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -726,10 +726,11 @@ following sections:
 Table statistics
 ^^^^^^^^^^^^^^^^
 
-Use :doc:`/sql/analyze` statements in Trino to populate the table statistics in
-Delta Lake. Data size and number of distinct values (NDV) statistics are
-supported; whereas minimum value, maximum value, and null value count statistics
-are not supported. The :doc:`cost-based optimizer
+Use :doc:`/sql/analyze` statements in Trino to populate data size and
+number of distinct values (NDV) extended table statistics in Delta Lake.
+The minimum value, maximum value, value count and null value count
+statistics are computed on the fly out of the *logical files* statistics
```

Comment:
I think users can't understand the meaning of "logical files". Can we rephrase that?

Reply:
Changed to _transaction log_ - it is a more generic notion and doesn't go into implementation details.